indico/modules/categories/templates/display/base.html

```diff
@@ -0,0 +1,121 @@
+                   title="{% trans %}View{% endtrans %}"
+                   data-toggle="dropdown"></a>
+                <ul class="dropdown">
+                    <li>
+                        <a href="{# TODO url_for('.today', category) #}">{% trans %}Today's events{% endtrans %}</a>
+                    </li>
+                    <li>
+                        <a href="{# TODO url_for('.week', category) #}">{% trans %}Week's events{% endtrans %}</a>
+                    </li>
+                    <li>
+                        <a href="{# TODO url_for('.calendar', category) #}">{% trans %}Calendar{% endtrans %}</a>
+                    </li>
+                    <li>
+                        <a href="{# TODO url_for('.map', category) #}">{% trans %}Category map{% endtrans %}</a>
+                    </li>
+                    <li>
+                        <a href="{{ url_for('.statistics', category) }}">{% trans %}Category statistics{% endtrans %}</a>
+                    </li>
+                </ul>
+                <a class="i-button icon-plus arrow js-dropdown"
+                   title="{% trans %}Create new event{% endtrans %}"
+                   data-toggle="dropdown"></a>
+                <ul class="dropdown">
+                    <li>
+                        <a href="{{ url_for('event_creation.conferenceCreation', event_type='lecture') }}">
+                            {% trans %}Lecture{% endtrans %}
+                        </a>
+                    </li>
+                    <li>
+                        <a href="{{ url_for('event_creation.conferenceCreation', event_type='meeting') }}">
+                            {% trans %}Meeting{% endtrans %}
+                        </a>
+                    </li>
+                    <li>
+                        <a href="{{ url_for('event_creation.conferenceCreation', event_type='conference') }}">
+                            {% trans %}Conference{% endtrans %}
+                        </a>
+                    </li>
+                </ul>
+                {% if category.can_manage(session.user) %}
+                    <a class="i-button icon-edit arrow js-dropdown"
+                       title="{% trans %}Management options{% endtrans %}"
+                       data-toggle="dropdown"></a>
+                    <ul class="dropdown">
+                        <li>
+                            <a href="{{ url_for('categories.manage_content', category) }}">
+                                {% trans %}Edit category{% endtrans %}
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#"
+                               data-ajax-dialog
+                               data-href="{{ url_for('.create_subcategory', category) }}"
+                               data-title="{% trans %}New category{% endtrans %}">
+                                {% trans %}Add subcategory{% endtrans %}
```

Comment:
Does it make much sense to keep this here? I'd rather have just a link to management and then you do it from there.

cc @pferreir